Kernel/Devices/BXVGADevice.cpp

```diff
@@ -75,7 +75,7 @@ u32 BXVGADevice::find_framebuffer_address()
     static const PCI::ID bochs_vga_id = { 0x1234, 0x1111 };
     static const PCI::ID virtualbox_vga_id = { 0x80ee, 0xbeef };
     u32 framebuffer_address = 0;
-    PCI::enumerate_all([&framebuffer_address](const PCI::Address& address, PCI::ID id) {
+    PCI::Access::the().enumerate_all([&framebuffer_address](const PCI::Address& address, PCI::ID id) {
```

Comment:
Instead of changing this everywhere, could we just have a convenient helper function in the `PCI` namespace:

```cpp
namespace PCI {
void enumerate_all(Function<void(Address, ID)> callback)
{
    PCI::Access::the().enumerate_all(callback);
}
```
This way we don't have to change all the code, and it abstracts away the `Access` concept from code that doesn't need to care about it anyway :)

Reply:
Agree, and I thought about it but I didn't implement it in the end (don't remember why). Anyway, that's a good idea :)